src/main/java/com/spotify/github/v3/clients/OrganisationClient.java

```diff
@@ -52,4 +63,28 @@ public TeamClient createTeamClient() {
   public GithubAppClient createGithubAppClient() {
     return new GithubAppClient(github, org);
   }
+
+  /**
+   * Get an org membership of a user.
+   *
+   * @param username username of the org member
+   * @return membership
+   */
+  public CompletableFuture<OrgMembership> getOrgMembership(final String username) {
+    final String path = String.format(MEMBERSHIP_TEMPLATE, org, username);
+    log.debug("Fetching org membership for: " + path);
+    return github.request(path, OrgMembership.class);
+  }
+
+  /**
+   * Add or update an org membership for a user.
+   *
+   * @param request update org membership request
+   * @return membership
+   */
+  public CompletableFuture<OrgMembership> updateOrgMembership(final OrgMembershipCreate request, final String username) {
+    final String path = String.format(MEMBERSHIP_TEMPLATE, org, username);
+    log.debug("Updating membership in org: " + path);
+    return github.put(path, github.json().toJsonUnchecked(request), OrgMembership.class);
```

Comment:
Is there a reason we're using json unchecked here?

Reply:
Honestly was just following what was already being used in other tests.